test/e2e/utils.py

```diff
@@ -168,6 +176,8 @@ def submit_job_from_bundle(
         create_job_args["job_parameters"] = job_parameters  # type: ignore
 
     job_id = create_job_from_job_bundle(**create_job_args)  # type: ignore
+    if debug_snapshot_dir:
+        return  # type: ignore
```

Comment:
I don't think we should `type: ignore` this. By returning `None`, callers do need to handle the case where there is no return value.